static/app/views/settings/project/projectTeams.tsx

```diff
@@ -12,26 +12,27 @@ import Pagination from 'sentry/components/pagination';
 import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import {t, tct} from 'sentry/locale';
 import TeamStore from 'sentry/stores/teamStore';
-import type {Organization} from 'sentry/types/organization';
-import type {Project} from 'sentry/types/project';
 import {decodeScalar} from 'sentry/utils/queryString';
 import routeTitleGen from 'sentry/utils/routeTitle';
 import {useLocation} from 'sentry/utils/useLocation';
 import {useNavigate} from 'sentry/utils/useNavigate';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
+import useProjects from 'sentry/utils/useProjects';
 import SettingsPageHeader from 'sentry/views/settings/components/settingsPageHeader';
 import TeamSelectForProject from 'sentry/views/settings/components/teamSelect/teamSelectForProject';
 import TextBlock from 'sentry/views/settings/components/text/textBlock';
 import {ProjectPermissionAlert} from 'sentry/views/settings/project/projectPermissionAlert';
 
-type ProjectTeamsProps = {
-  organization: Organization;
-  project: Project;
-};
-
-function ProjectTeams({organization, project}: ProjectTeamsProps) {
+export default function ProjectTeams() {
   const navigate = useNavigate();
   const location = useLocation();
 
+  const organization = useOrganization();
+  const {projectId} = useParams<{projectId: string}>();
+  const {projects} = useProjects({slugs: [projectId]});
+  const project = projects.find(p => p.slug === projectId)!;
```

Comment:
so i think in this case we want to refactor the parent component to pass the project down via the Outlet context to keep the behavior the same as the old behavior

Reply:
useOrganization is fine though, just in project settings we want to access the project in the same way as before.

Reply:
This would be a larger refactor and likely need to touch multiple project settings pages.

Reply:
the project objects are potentially different since ProjectContext loads things and slams it into a local state

Reply:
do not trust the `Project` type in general. The Project from the store will be missing fields that are only available when calling the full project from the details endpoint